.github/workflows/test.yml

```diff
@@ -10,45 +10,44 @@ on:
     - cron: "0 1 * * *"
   workflow_dispatch:
 jobs:
-  checks:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
-        with:
-          python-version: 3.8
-      - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install ".[dev]"
-      - name: Audit Python packages for known vulnerabilities
-        run: |
-          if ! RES=$(ossaudit --installed --config setup.cfg); then
-            RES="${RES//'%'/'%25'}"
-            RES="${RES//$'\n'/'%0A'}"
-            RES="${RES//$'\r'/'%0D'}"
-            echo "::error ::${RES}"
-            exit 1
-          fi
-          echo ${RES}
-      - name: Format with black
-        run: |
-          if ! RES=$(black --check $(git diff --name-only --diff-filter=AM master -- "*.py") 2>&1); then
-            RES="${RES//'%'/'%25'}"
-            RES="${RES//$'\n'/'%0A'}"
-            RES="${RES//$'\r'/'%0D'}"
-            echo "::error ::${RES}"
-            exit 1
-          fi
-          echo ${RES}
+  # checks:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Set up Python 3.8
+  #       uses: actions/setup-python@v1
+  #       with:
+  #         python-version: 3.8
+  #     - name: Checkout code
+  #       uses: actions/checkout@v2
+  #     - name: Install dependencies
+  #       run: |
+  #         python -m pip install --upgrade pip
+  #         pip install ".[dev]"
+  #     - name: Audit Python packages for known vulnerabilities
+  #       run: |
+  #         if ! RES=$(ossaudit --installed --config setup.cfg); then
+  #           RES="${RES//'%'/'%25'}"
+  #           RES="${RES//$'\n'/'%0A'}"
+  #           RES="${RES//$'\r'/'%0D'}"
+  #           echo "::error ::${RES}"
+  #           exit 1
+  #         fi
+  #         echo ${RES}
+  #     - name: Format with black
+  #       run: |
+  #         if ! RES=$(black --check $(git diff --name-only --diff-filter=AM master -- "*.py") 2>&1); then
+  #           RES="${RES//'%'/'%25'}"
+  #           RES="${RES//$'\n'/'%0A'}"
+  #           RES="${RES//$'\r'/'%0D'}"
+  #           echo "::error ::${RES}"
+  #           exit 1
+  #         fi
+  #         echo ${RES}
   unit-tests:
-    needs: checks
     runs-on: ubuntu-latest
```

Comment:
I'd specify a version of ubuntu we know works (like 20.04); it shouldn't influence this particular repo, but we've had issues with 22.04 and mongo as the package maintainer was behind and didn't have packages set up for it